Can't build UI

When I try to build the UI with `npm run build`, the following error appears:
```
> readflow-app@0.1.0 build
> npm run lint && react-scripts build


> readflow-app@0.1.0 lint
> eslint src/**/*.{js,jsx,ts,tsx,json}


Oops! Something went wrong! :(

ESLint: 8.25.0

No files matching the pattern "src/**/*.jsx" were found.
Please check for typing mistakes in the pattern.
```

> Hi, I am confused because I am unable to reproduce your issue. Can you retry on a fresh clone using the `make ui` command?
> 
> ```shell
> git clone git@github.com:ncarlier/readflow.git
> cd readflow
> git submodule init && git submodule update
> make ui
> ```

Still same issue
```
make: go: No such file or directory
>>> Building Web UI...

> readflow-app@0.1.0 build
> npm run lint && react-scripts build


> readflow-app@0.1.0 lint
> eslint src/**/*.{js,jsx,ts,tsx,json}


Oops! Something went wrong! :(

ESLint: 8.25.0

No files matching the pattern "src/**/*.jsx" were found.
Please check for typing mistakes in the pattern.

make: *** [Makefile:132: ui] Error 2
```

--- a/ui/package.json
+++ b/ui/package.json
@@ -44,7 +44,7 @@
     "gitpod": "GENERATE_SOURCEMAP=false REACT_APP_AUTHORITY=none REACT_APP_API_ROOT=https://8080-${GITPOD_WORKSPACE_ID}.${GITPOD_WORKSPACE_CLUSTER_HOST} react-scripts start",
     "build": "npm run lint && react-scripts build",
     "test": "react-scripts test",
-    "lint": "eslint src/**/*.{js,jsx,ts,tsx,json}"
+    "lint": "eslint --ext .js,.jsx,.ts,.tsx src/"
   },
   "eslintConfig": {
     "extends": "react-app"

